tools/dreport.d/plugins.d/inventory

```diff
@@ -17,11 +17,9 @@ command="busctl call --verbose --no-pager \
 file_name="inventory.log"
 
 add_cmd_output "$command" "$file_name" "$desc"
-
-retval=$?
-if [ $retval -ne 0 ]; then
+if [ $? -ne 0 ]; then
```

Comment:
it is separate change, please push a separate commit

Reply:
this changed is added as part of the script so any code commet I should be doing in the same commit

Reply:
I did not get it, it is a separate script but, I will leave it to you - just suggestion comment

Reply:
agree ramesh sorry this github is bit confusing, i will see that code goes back to original place